tests/unit_tests/test_material.py

```diff
@@ -3,7 +3,7 @@
 import openmc.stats
 import openmc.examples
 import pytest
-
+from collections import defaultdict
```

Comment:
PEP8: Order of imports should be 1) standard library, 2) thick-party, 3) local, each separated by a blank line. Can you move this above the openmc/pytest imports?

Reply:
Moved the import as suggested and imports now follow the specified order